app/views/admin/collections/show.html.erb

```diff
@@ -51,7 +51,8 @@ Unless required by applicable law or agreed to in writing, software distributed
       <% end %>
       <% if @collection.website_url.present? %>
         <p><strong>Website:</strong></p>
-        <p><%= link_to(@collection.website_label, @collection.website_url)%></p>
+        <% website_label = @collection.website_label.present? ? @collection.website_label : @collection.website_url %>
```

Comment:
Would this be cleaner? `website_label = @collection.website_label || @collection.website_url`
It can be merged with the next line too.

Reply:
We need the `present?` check because it could be empty string.  Or we could try to handle that in the model or controller somewhere by turning `""` into `nil`.

Reply:
I see. Looks like `presence` will handle that https://apidock.com/rails/Object/presence
i.e. `@collection.website_label.presence || @collection.website_url`

Reply:
Neat!  I've never seen that before.